pymarketstore/grpc_client.py

```diff
@@ -23,7 +23,12 @@ class GRPCClient(object):
 
     def __init__(self, endpoint: str = 'localhost:5995'):
         self.endpoint = endpoint
-        self.channel = grpc.insecure_channel(endpoint)
+        # set max message sizes
+        options = [
+            ('grpc.max_send_message_length', 1 << 31 - 1),
+            ('grpc.max_receive_message_length', 1 << 31 - 1),
```

Comment:
Exactly 1GB, cool... curious why not `1 << 30`? (or did you mean `(1 << 31) - 1` for 2GB minus 1byte?)

Reply:
Also TIL, at least in Python, using shift is barely faster than `1 * 1024**3`, which might be more readable/obvious for non-low-level devs?
```
%timeit 1 << 30
4.46 ns ± 0.0184 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

%timeit 2**30
4.47 ns ± 0.0148 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)

%timeit 1 * 1024**3
4.49 ns ± 0.0172 ns per loop (mean ± std. dev. of 7 runs, 100000000 loops each)
```

Reply:
sorry, I just copy-pasted the value from `math.MaxInt32` in marketstore code, but I agree that `1* 1024**3` should be much more readable. Let me change it...